LayoutTests/fast/scrolling/mac/scrollbars/scrollbars-controller-crash.html

```diff
@@ -0,0 +1,17 @@
+Test passes if there is no assertion or crash.
+
+<script>
+function freememory() {
+}
+function eventhandler2() {
+try { document.write("1");; } catch(e) { }
+}
+</script>
+<details open="true" style="?{#Se&amp;eI9c?~DvW" ontoggle="eventhandler2()" pseudoElement="}G)`2;c)E%%)ZJ">
```

Comment:
It seems like this test could be reduced further.

Reply:
Fixed.